contracts/contracts/token/WOETH.sol

```diff
@@ -106,87 +113,8 @@ contract WOETH is ERC4626, Governable, Initializable {
 
     /** @dev See {IERC4262-totalAssets} */
     function totalAssets() public view override returns (uint256) {
```

Comment:
We could remove the todo comment in transferToken?

Reply:
Agreed: https://github.com/OriginProtocol/origin-dollar/pull/2453/commits/ca1a9984fb2481f3267bf507dd57527aa1f1825e

We don't need to think about doing this. Lets implement it once someone actually donates a substantial amount